.github/workflows/update.yml

```diff
@@ -0,0 +1,60 @@
+name: Update README and LaTeX Build File
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-files:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      
+      - name: Update files with Python
+        run: |
+          python -c "\
```

Comment:
Let's put this in an actual python file. This is more readable and maintainable compared to this inline code